exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusMetricReader.java

```diff
@@ -29,11 +29,14 @@ public class PrometheusMetricReader implements MetricReader, MultiCollector {
   private final Otel2PrometheusConverter converter;
 
   // TODO: refactor to public static create or builder pattern to align with project style
-  /** See {@link Otel2PrometheusConverter#Otel2PrometheusConverter(boolean, Predicate)}. */
+  /** See {@link Otel2PrometheusConverter#Otel2PrometheusConverter(boolean, Predicate, boolean)}. */
   public PrometheusMetricReader(
```

Comment:
Since this is a public constructor, even if the module isn't stable, let's create an overload constructor with all 3 parameters, and have this one delegate to it with utf8 turned off. Deprecate the 2-parameter one, and then when we stabilize, we can remove it.